src/Beberlei/Metrics/Collector/Null.php

```diff
@@ -15,6 +15,7 @@
 
 /**
  * @deprecated for PHP7 compatibility, use NullCollector instead.
+ * @todo Remove this class
```

Comment:
Because this TODO adds strictly no value: removing deprecated APIs can only be done in major versions, and the ``@deprecated`` tag already allows to locate them so there is no need for the TODO asking to remove the class.